tensorflow/inference/docker/build_artifacts/sagemaker/nginx.conf.template

```diff
@@ -15,7 +15,7 @@ http {
   include /etc/nginx/mime.types;
   default_type application/json;
   access_log /dev/stdout combined;
-  js_include tensorflow-serving.js;
+  js_import tensorflow-serving.js;
```

Comment:
Please remove the `-` in `tensorflow-serving.js` as js_import expects not to have `-` in the header file name